### ISSUE TITLE
Policy generator uses incorrect SQS permissions for *Batch actions

### ISSUE THREAD

**Issue by xavierbuspatrol** [aws/chalice#1858]
Chalice version 1.26.2
Python 3.9

TL;DR - Policy generator will output IAM policies with `sqs:SendMessageBatch` or `sqs:DeleteMessageBatch`, even though it is not a valid IAM permission.

Steps to reproduce:

1. Create a simple chalice app that uses ```sqs_client.send_message_batch``` and/or ```sqs_client.delete_message_batch``` (See here: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.send_message_batch)
2. Use `chalice gen-policy`
3. See that the IAM auto-generated policy contains IAM permissions that do not exist. Example:
```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "sqs:DeleteMessageBatch",
        "sqs:GetQueueUrl",
        "sqs:ReceiveMessage",
        "sqs:SendMessageBatch"
      ],
      "Resource": [
        "*"
      ],
      "Sid": "eef17f7dab3841c09ab195fa2533cef5"
    }
  ]
}
```

`sqs:SendMessageBatch` and `sqs:DeleteMessageBatch` are valid SQS actions, but are not valid IAM permissions. It should be `sqs:SendMessage` and `sqsDeleteMessage`. See the table of required permissions for actions in SQS: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-api-permissions-reference.html

**Comment by jamesls** [comment]
Thanks for reporting, we'll get that updated.
